cookbooks/aws-parallelcluster-slurm/templates/default/slurm/slurm.conf.erb

```diff
@@ -40,19 +40,6 @@ SuspendTimeout=120
 PrivateData=cloud
 ResumeRate=0
 SuspendRate=0
-<% if node["cluster"]["add_node_hostnames_in_hosts_file"] == 'true' -%>
-#
-# PROLOG AND EPILOG
-# prolog is executed to add nodes info to /etc/hosts on compute nodes when each job is allocated
-# epilog is executed to clean contents written by prolog
-# PrologFlags specifies the prolog is executed at job allocation and prologs and epilogs are of different jobs are executed serially
-# SchedulerParameters allows jobs to be requeued to other nodes if prolog error exits.
-# Note the error exit of prolog drains a node, because the error of prolog is considered as a node error.
-Epilog=<%= node['cluster']['slurm']['install_dir'] %>/etc/pcluster/epilog.d/*
-Prolog=<%= node['cluster']['slurm']['install_dir'] %>/etc/pcluster/prolog.d/*
-PrologFlags=alloc,serial
-SchedulerParameters=nohold_on_prolog_fail
-<% end -%>
```

Comment:
If the decision to remove this is independent of the Slurm upgrade, I would split the commit in two (we can keep the commits in the same PR if you wish).

Reply:
Done